libarchive/ffi.py

```diff
@@ -127,9 +127,12 @@ def ffi(name, argtypes, restype, errcheck=None):
 ffi('entry_rdevmajor', [c_archive_entry_p], c_uint)
 ffi('entry_rdevminor', [c_archive_entry_p], c_uint)
 
-ffi('entry_set_size', [c_archive_entry_p, c_longlong], c_int)
+ffi('entry_set_size', [c_archive_entry_p, c_longlong], c_longlong)
```

Comment:
yes, this change is necessary to allow compressing big files (our files are huge).
BTW longlong is the type that actually matches the C signature.